description.md

```diff
@@ -38,6 +38,73 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Assignments
 
+Assignments are unassisted practical activities that learners undertake individually or in teams over a longer timespan.
+The primary objective of an assignment is to create a compelling scenario that allows learners to apply the concepts they have learned.
```

Comment:
I have used the term "concept", however, @razvand suggested that "topic" is a better term. I have updated my PRs to use "topic", so you should also do it for the sake of consistency.

Reply:
Also, I would emphasize the fact the difference between assignments and drills/quizzes. Note that some universities have assignments in the form of a collection of quizzes or drills or both.